### PR TITLE
Rhyen sftware engr patch 1

### PR DESCRIPTION
**Pull request by SollorinJohnMichael**
Hi @RhyenSftwareEngr , please review my newly added file

### CODE REVIEW

**Comment by RhyenSftwareEngr** [comment]
Hi @SollorinJohnMichael  It seems that you did not use the correct datatype

File: ArithmeticOperations [hunk 1]
```diff
@@ -0,0 +1,24 @@
+public class ArithmeticOperations{ 
+
+  var numOne: Double
+  var numTwo: Double
+  var result: Double
+  
+  func add(numOne: Double, numTwo: Double) -> double {
```
Comment (RhyenSftwareEngr) [inline ArithmeticOperations:7]: It seems that you used a wrong datatype here

**Review by RhyenSftwareEngr — APPROVED** [review]
Kudos

**Comment by SollorinJohnMichael** [comment]
okay

**Pull request closed without merging**